### ISSUE TITLE
YOLOv7-tiny

### ISSUE THREAD

**Issue by MonsieurMiku** [bubbliiiing/yolov7-pytorch#21]
您好，YOLOv7-tiny会考虑添加嘛

**Comment by bubbliiiing** [comment]
官方现在还没给权值。

**Comment by wzp8023391** [comment]
我也对这个tiny版本非常感兴趣，官网貌似给了

**Comment by bubbliiiing** [comment]
现在只给了config吧
![image](https://user-images.githubusercontent.com/47347516/182030196-01029487-8943-48fd-ac9c-940b88a3fd89.png)

**Comment by MonsieurMiku** [comment]
release里面有个yolov7-tiny.pt      https://github.com/WongKinYiu/yolov7/releases

**Comment by bubbliiiing** [comment]
有空做一下，最近比较忙

**Comment by caidupeng** [comment]
泡泡哥，yolov7-tiny大概什么时候能出啊

**Comment by bubbliiiing** [comment]
出了

**Comment by maomao123-git** [comment]
我能弱弱说一句，v7tiny效果没有v5好吗？自己的数据集。而且差的挺远，收敛是很快，但是，val loss 最低的权值，效果很差

**Comment by bubbliiiing** [comment]
谁的库呢
